### ISSUE TITLE
Remove 'Accessible Features For Residents With Disabilities?' filter

### ISSUE THREAD

**Issue by jnowell** [CityOfDetroit/bloom#916]
Since we've added the new 'Accessibilty Features' filters, that allow users to filter off of 12 different accessibility-related features, the subsquent dropdown in the filter modal titled 'Accessible Features For Residents With Disabilities?' should be removed, as it adds no value.

![accessibility_features_filter](https://user-images.githubusercontent.com/394908/147594710-632f4a32-b50e-44c2-9af8-5c8c40ddd820.png)
